### ISSUE TITLE
iOS 9.1 simulator

### ISSUE THREAD

**Issue by SidharthPHariharan** [wymsee/cordova-HTTP#90]
After enabling ssl pinning I tried a get with cordovaHTTP, it is going to error response callback with response.error return empty. Can someone help me with this?

**Comment by gsccheng** [comment]
Yeah, for me, the error `response` does not follow the type in the documentation. It should have `error` and `status` properties but I only get `{error: ""}`. I can put in any url and I'll get the same message. I also wasn't able to get `enableSSLPinning` to fail, even when I don't include any certs at all.
